pkg/pipelineascode/pipelineascode.go

```diff
@@ -160,6 +160,7 @@ func Run(ctx context.Context, cs *cli.Clients, k8int cli.KubeInteractionIntf, ru
 		"tekton.dev/pipeline-ascode-sender":     runinfo.Sender,
 		"tekton.dev/pipeline-ascode-event-type": runinfo.EventType,
 		"tekton.dev/pipeline-ascode-branch":     refTomakeK8Happy,
+		"tekton.dev/pipeline-ascode-reporef":    repo.GetName(),
```

Comment:
How about naming it as `tekton.dev/pipeline-ascode-repo-name`?

Reply:
one more suggestion `tekton.dev/pipeline-ascode-owner`: `<repository_cr_name>`

Reply:
i think "tekton.dev/pipeline-ascode-repo-name" may be better, owner may be confused with actual github repo owner,